test/lib/smart-survey-api.test.js

```diff
@@ -8,15 +8,18 @@ let getResponsesStub = sinon.stub();
 const smartSurveyStub = function() {
     return {getResponses: getResponsesStub};
 };
-// Proxyquire syntax: first argument: point to your module (which holds your dependency)
-// 2nd argument: An object with the key: module name to proxy & the value the object above
+// Proxyquire syntax:
+// first argument: points to your module (which holds your dependency)
+// 2nd argument: An object. The object has a key that is the name of the moduleof the dependency
+// & the value the object above of the proxy
 // So this means smartSurveyStub will now point to this function, i.e. a stub
-const smartSurveyAPI = proxyquire('../../lib/smart-survey-api', { 'smartsurvey-client': smartSurveyStub});
+const SmartSurveyAPIBase = proxyquire('../../lib/smart-survey-api', { 'smartsurvey-client': smartSurveyStub});
+const smartSurveyAPI = new SmartSurveyAPIBase('mytoken', 'mytokensecret');
```

Comment:
It came out of feedback from @oliverweighell It was just to make it a better design and reusable.  @oliverweighell what are your thoughts